Readme.md

```diff
@@ -16,38 +16,44 @@ A simple app to practice Hebrew verbs.
 
 ## Configuration
 
-First, create a MySQL database and a user that can connect to it. In this example, we'll use `hebrew_db`, `hebrew_user` and `hebrew_pass`.
+Copy the `.env.example` file in the root directory and call it `.env` and customise it.
 
-Create a `.env` file in the root directory with at least the following options:
+Install `npm` packages:
 
-    APP_ENV=production
-    APP_DEBUG=false
-    APP_KEY= # a 32-char random string
-    APP_URL=https://demo.camilstaps.nl/HebrewParseTrainer/ # e.g., use your own URL here
+    $ npm install
 
-    APP_LOCALE=en
-    APP_FALLBACK_LOCALE=en
+Run the development server:
 
-    DB_CONNECTION=mysql
-    DB_HOST=localhost
-    DB_PORT=3306
-    DB_DATABASE=hebrew_db
-    DB_USERNAME=hebrew_user
-    DB_PASSWORD=hebrew_pass
+    $ npm run watch
 
-    CACHE_DRIVER=memchached
-    SESSION_DRIVER=memcached
-    QUEUE_DRIVER=database
+### Docker Compose
 
-Return to the root directory and run:
+To initialise the database:
 
+    $ docker compose run --rm app bash -c 'composer install && php artisan migrate --seed'
```

Comment:
Here and below, should this be `docker-compose` with hyphen instead of space, or am I running an outdated version?

Reply:
Yes, you are probably running a v1 version of Compose, I think they changed it in v2. Since v1 is no longer updated or distributed, I think using the v2 syntax is best to use going forward.